app/src/main/java/timur/gilfanov/messenger/ui/screen/settings/SettingsSideEffects.kt

```diff
@@ -0,0 +1,8 @@
+package timur.gilfanov.messenger.ui.screen.settings
+
+import timur.gilfanov.messenger.domain.usecase.user.repository.ObserveSettingsRepositoryError
+
+sealed interface SettingsSideEffects {
+    data object Unauthorized : SettingsSideEffects
+    data class GetSettingsFailed(val error: ObserveSettingsRepositoryError) : SettingsSideEffects
```

Comment:
Must rename to be constant with operation that cause the side-effect.
```suggestion
    data class ObserveSettingsFailed(val error: ObserveSettingsRepositoryError) : SettingsSideEffects
```